### DISCUSSION TITLE
VS Code Server "Can't Install on Proxmox"

### DISCUSSION THREAD

**Discussion by censey** [tteck/Proxmox#2836] [Q&A] [answered]
Getting a general error when it pulls the version of the host during script execution.  Do I need to run this helper script in a different host OS?

**Comment by tteck** [comment, marked as answer]
![Screenshot 2024-04-22 10 06 19 AM](https://github.com/tteck/Proxmox/assets/21342670/696c6249-a45d-482a-a36d-d2ed0e61bb60)

**Reply by wohfab** [reply]
Could someone explain what "_the_ LXC console" is supposed to be? I don't find anything about it in the docs.

**Reply by alain-br** [reply]
I have the same question; how do you access the LXC console?  I can see "_Console" in an existing LXC, so does that mean you need to create a generic LXC first, and the run the required command? Thank you.
